.pipelines/ci.yml

```diff
@@ -473,6 +473,70 @@ stages:
         python -m pytest --ignore=test_cv2.py --ignore=test_tools_add_pre_post_processing_to_model.py . --verbose
       displayName: Run python test
 
+- stage: WindowsCUDABuilds
+  dependsOn: []
+  jobs:
+  - job: WindowsC
+    pool:
+      name: 'onnxruntime-extensions-Win2022-GPU-A10'
+    variables:
+      ort.version: '1.16.1'
+    steps:
+      - template: templates/set_winenv.yml
+        parameters:
+          EnvSetupScript: 'set_env_cuda.bat'
+          DownloadCUDA: true
+
+      - script: |
+          nvidia-smi
+          nvcc --version
+          where nvcc
+        displayName: check cuda version
+
+      - task: DownloadGitHubRelease@0
```

Comment:
Can we get it from Nuget instead? Then we can use a prerelease version when needed. We have a https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly feed.  It would be great if we can test every ORT release with this pipeline before the release is published.

Reply:
Like, first the release manager publish a test package to the ADO feed. Then we manually run this pipeline and specify the test package's version number when we queue the build.

Reply:
Same thought, but since they are unstable and the fixing could take some time. I'd like to create an CI.optional pipeline to put this job, as well as Androids.

Reply:
We can start with using a stable release from nuget.org first. Then we may add another pipeline to build ort-extension with a nightly ORT release.